### ISSUE TITLE
Codegen timeline

### ISSUE THREAD

**Issue by beau-midigator** [sourcey/spectacle#52]
I noticed the placeholders in the html for the codegen (curl, python, ruby, etc) that would mimic stripe.  Is that a long term goal?

**Comment by auscaster** [comment]
Yes it would be great to add this feature, although I don't have a timeline for it myself. If you would like to have a go at a PR I'm sure it would be gratefully received by spectacle users!

**Comment by auscaster** [comment]
Closing as superseded. This repo has been rewritten; `spectacle-docs` is no longer maintained and now hosts [Sourcey](https://sourcey.com), a static docs platform on Astro + Preact. If it still bites in the new build, please file fresh.
